hyperspy/_signals/hologram_image.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2016 The HyperSpy developers
+#
+# This file is part of  HyperSpy.
+#
+#  HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from hyperspy._signals.image import Image
+
+
+class HologramImage(Image):
+    """Image subclass for holograms acquired via electron holography."""
+

```

Comment:
Perhaps it wasn't clear - but this was precisely the point behind my suggestion that "I think it's best to add a bit more to this PR e.g. some methods in HologramImage"... A separate PR would also be fine obviously .

My main point was more that we could split off bits that aren't specifically holography related and can be reviewed by anyone and then have one PR with the really holographic bits in for someone more expert to review.